mods/ra/rules/campaign-rules.yaml

```diff
@@ -44,3 +44,13 @@ E7.noautotarget:
 
 ^Vehicle:
 	Demolishable:
+
+FCOM:
+	ExternalCapturable:
```

Comment:
This will conflict with #15661

Reply:
I'll wait for that PR and then rebase this one.